.github/workflows/build.yaml

```diff
@@ -40,7 +40,10 @@ jobs:
       with:
         fetch-depth: 1
     - name: Installing dependencies
-      run: sh ./.github/scripts/install_deps.sh
+      run: |
+        # https://github.com/actions/runner-images/issues/7192
+        echo 'APT::Get::Always-Include-Phased-Updates "false";' | sudo tee /etc/apt/apt.conf.d/99-phased-updates
+        sh ./.github/scripts/install_deps.sh
```

Comment:
Shouldn't the workaround be put in `install_deps.sh` in a single place?

But my vote goes to removing `apt-get upgrade` anyway. :)

Reply:
Sorry, I was a bit overacting here.
And with your other comment, I think you are right. So let's remove it and see how long it works ;)